client/src/components/BuildDetails.js

```diff
@@ -1,53 +1,61 @@
 import React from 'react';
 import { connect } from 'react-redux';
-
-function Build({builderid, number, builderName, started_at, complete_at, state_string}) {
-  return (<React.Fragment>
-    <div className='col-sm-3'>
-      <a
-        target='_blank'
-        rel='noreferrer noopener'
-        href={`https://build.reactos.org/#builders/${
-          builderid
-        }/builds/${number}`}
-      >
-        {builderName}
-      </a>
-    </div>
-    <div className='col-sm-3'>
-      {state_string}
-      {state_string === 'build successful' ? (
-        <i className='fa fa-check' />
-      ) : (
-        <i />
-      )}
-    </div>
-    <div className='col-sm-3'>started_at:{started_at}</div>
-    <div className='col-sm-3'>complete_at:{complete_at}</div>
-  </React.Fragment>);
+function Build({ build, builderName }) {
+  let completedDate = new Date(build.complete_at * 1000);
+  let startedDate = new Date(build.started_at * 1000);
+  return (
+    <React.Fragment>
+      <div className='col-sm-2'>
+        <a
+          target='_blank'
+          rel='noreferrer noopener'
+          href={`https://build.reactos.org/#builders/${
+            build.builderid
+          }/builds/${build.number}`}
+        >
+          {builderName}
+        </a>
+      </div>
+      <div className='col-sm-3'>
+        {build.state_string}
+        {build.state_string === 'build successful' ? (
+          <i className='fa fa-check' />
+        ) : (
+          <i />
+        )}
+      </div>
+      <div className='col-sm-3'>Started: {startedDate.toLocaleString()}</div>
+      <div className='col-sm-4'>
+        Completed: {build.complete_at ? completedDate.toLocaleString() : <p />}
+      </div>
+    </React.Fragment>
+  );
 }
 
 function renderBuild(props) {
-  return <Build key={props.buildid} {...props} />;
+  return <Build key={props.build.buildid} {...props} />;
 }
 
-function BuildDetails({builds}) {
-  return (<React.Fragment>
-    {builds.length > 0 ? (
-      <div className='row'>{builds.map(renderBuild)}</div>
-    ) : (
-      <p>
-        <strong>No data Exists</strong>
-      </p>
-    )}
-  </React.Fragment>);
+function BuildDetails({ builds }) {
+  return (
+    <React.Fragment>
+      {builds.length > 0 ? (
+        <div className='row'>{builds.map(renderBuild)}</div>
+      ) : (
+        <p>
+          <strong>No data Exists</strong>
+        </p>
+      )}
+    </React.Fragment>
+  );
 }
 
 const mapStateToProps = ({ builders }, ownProps) => {
   return {
-    builds: ownProps.builds.map(
-      build => {...build, builderName: builders[build.builderid].name}
-    )
+    builds: ownProps.builds.map(build => ({
+      build,
```

Comment:
Why you removed the spread operator? It's a standard JS feature (ES 2018)

Reply:
looks like my plugin has messed up.. i wasn't aware of this change. I'll correct it